CMakeLists.txt

```diff
@@ -109,6 +109,38 @@ IF(BUILD_TESTS)
   INCLUDE("${CMAKE_CURRENT_LIST_DIR}/test/CMakeLists.txt")
 ENDIF(BUILD_TESTS)
 
+################################
```

Comment:
If we stick to this approach, we could put this part into a separate cmake file. And rename the folder to cmake instead of cmake_modules as the files there are not only modules.